ansible/memcached-operator/README.md

```diff
@@ -1,46 +1,105 @@
-# Memcached Ansible Operator
+# Memcached Operator
 
-## Quickstart
+## Overview
 
-1. Start [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) (e.g. `minikube start`)
+This Memcached operator is a simple example operator for the [Operator SDK][operator_sdk] and includes some basic end-to-end tests.
 
-    We will be using the `default` namespace, and the `default` account
+## Prerequisites
 
-1. Deploy the Memcached Ansible Operator
+- [dep][dep_tool] version v0.5.0+.
```

Comment:
I don't think we need these for the Ansible Operator, the binary is already built and in the base image.

Reply:
👍